genesyscloud/external_contacts_organization/data_source_genesyscloud_externalcontacts_organization.go

```diff
@@ -50,7 +50,7 @@ func hydrateOrganizationCacheFn(c *rc.DataSourceCache, ctx context.Context) erro
 
 	log.Printf("Hydrating cache for data source %s", ResourceType)
 
-	allExternalOrganization, resp, err := proxy.getAllExternalContactsOrganization(ctx, "")
+	allExternalOrganization, resp, err := proxy.getAllExternalContactsOrganization(ctx)
```

Comment:
@kavinbalagen Please make sure you use Amazon Q to do the following:

1.  Scan the functions you wrote for nil pointer exceptions.
2. Ask Amazon Q to document any functions you have touched your code
3. Ask Amazon Q for any suggested refactors